src/app/message/message.js

```diff
@@ -91,6 +91,9 @@ angular.module('app.message', [
   self.workers = workers;
   mediator.subscribeForScope('wfm:message:created', $scope, function(message) {
     message.sender = profileData;
+    //remove generated angular variables
+    message = JSON.parse(angular.toJson(message));
```

Comment:
JSON.parse(angular.toJson(message)); should be time consuming operation. We need find some other way to skip generated angular variables. @nialldonnellyfh WDYT?